integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -43,10 +55,463 @@ class AuthorizationServerIT extends ITSupport {
+
+    // Scope operations
+
+    @Test
+    void listOAuth2ScopesTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        OAuth2Scope oAuth2Scope = client.instantiate(OAuth2Scope)
+            .setName("java-sdk-authorization-server-scope" + RandomStringUtils.randomAlphanumeric(10))
+
+        OAuth2Scope createdOAuth2Scope = createdAuthorizationServer.createOAuth2Scope(oAuth2Scope)
+
+        assertThat(createdOAuth2Scope, notNullValue())
+        assertPresent(createdAuthorizationServer.listOAuth2Scopes(), createdOAuth2Scope)
+    }
+
+    @Test
+    void getOAuth2ScopesTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        OAuth2Scope oAuth2Scope = client.instantiate(OAuth2Scope)
+            .setName("java-sdk-authorization-server-scope" + RandomStringUtils.randomAlphanumeric(10))
+
+        OAuth2Scope createdOAuth2Scope = createdAuthorizationServer.createOAuth2Scope(oAuth2Scope)
+
+        assertThat(createdOAuth2Scope, notNullValue())
+
+        OAuth2Scope retrievedOAuth2Scope = createdAuthorizationServer.getOAuth2Scope(createdOAuth2Scope.getId())
+
+        assertThat(retrievedOAuth2Scope.getId(), equalTo(createdOAuth2Scope.getId()))
+    }
+
+    @Test
+    void updateOAuth2ScopesTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        OAuth2Scope oAuth2Scope = client.instantiate(OAuth2Scope)
+            .setName("java-sdk-authorization-server-scope" + RandomStringUtils.randomAlphanumeric(10))
+            .setConsent(OAuth2Scope.ConsentEnum.REQUIRED)
+            .setMetadataPublish(OAuth2Scope.MetadataPublishEnum.ALL_CLIENTS)
```

Comment:
is this `ALL_CLIENTS` the same as the above string?
Can it be reused when the enum is pulled out of this class?

Reply:
Yes, its the same. 

`public enum MetadataPublishEnum {
    ALL_CLIENTS("ALL_CLIENTS"),
    NO_CLIENTS("NO_CLIENTS");
`

Not sure where it can be reused as I do not see any other place where its used.

Reply:
Can you link a GitHub issue/PR for this one, so it's in the thread?

Reply:
https://oktainc.atlassian.net/browse/OKTA-305898 will address this.